script/update-typedoc-link.js

```diff
@@ -32,12 +32,12 @@ walk('docs')
     });
   });
 
-fs.readFile('docs/assets/js/search.js', 'utf8', (err, data) => {
+fs.readFile('docs/assets/js/search.json', 'utf8', (err, data) => {
   if (err) return console.error(err);
 
   let result = data.replace(/["][,]["]/g, '",\n"');
   result = result.replace(/[}][,][{]/g, '},\n{');
-  fs.writeFile('docs/assets/js/search.js', result, 'utf8', err => {
+  fs.writeFile('docs/assets/js/search.json', result, 'utf8', err => {
```

Comment:
Could you explain how does this work?

Reply:
[The fix is in `package.json`](https://github.com/aws/amazon-chime-sdk-js/blob/master/package.json#L27) — we add an explicit reference to the main `amazon-chime-sdk-js` repo as a command-line argument, so everywhere that builds now does so assuming a PR to the main repo.

```
    "doc": "node node_modules/typedoc/bin/typedoc --options typedoc.json --gitRemote https://github.com/aws/amazon-chime-sdk-js",
```

(I tried to add that as a config option in the typedoc config file and it complained, but the arg worked.)

This change just tracks a change in typedoc, which needed to upgrade to gain that command-line argument:

https://github.com/TypeStrong/typedoc/issues/1247#issuecomment-616291123